setup.py

```diff
@@ -1,7 +1,7 @@
 from distutils.core import setup
 setup(
-  name = 'coala-decorators',
-  packages = ['coala-decorators'],
+  name = 'coala_decorators',
```

Comment:
Note that the name can still have hyphen.
For example, the coala-bears repo has `name = "coala-bears"` and `packages = ["bears"]`
Again, This is your pereference, so I'm going to ack it anyway